docs/k8s/crds/agentendpoint.mdx

```diff
@@ -387,3 +387,185 @@ spec:
   - name: my-client-cert
     namespace: default
 ```
+
+## Status
+
+The `status` field contains the current state of the `AgentEndpoint` as reported by the ngrok operator.
+
+**Type:** `Object`
+
+**Fields:**
+
+| Field Name                                     | Type       | Description                                                                                                                           |
+| ---------------------------------------------- | ---------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| [`status.assignedURL`](#statusassignedurl)     | `string`   | The assigned URL. This will either be the user-supplied url, or the generated assigned url depending on the configuration of spec.url |
```

Comment:
Note from @nijikokun , we can change this to `assignedUrl` to match k8s common practices